packages/fxa-auth-server/lib/payments/capability.ts

```diff
@@ -843,4 +846,66 @@ export class CapabilityService {
     }
     return clientsFromStripe;
   }
+
+  /**
+   * Fetch the list of capabilities for the given plan ids
+   */
+  async planIdsToClientCapabilities(
+    subscribedPrices: string[]
+  ): Promise<ClientIdCapabilityMap> {
+    const stripeCapabilities = await this.planIdsToClientCapabilitiesFromStripe(
+      subscribedPrices
+    );
+
+    if (!this.capabilityManager) {
+      if (
+        this.logToSentry(
+          'planIdsToClientCapabilities.CapabilityManagerNotFound'
+        )
+      ) {
+        Sentry.captureMessage(
+          `CapabilityManager not found.`,
+          'error' as SeverityLevel
+        );
+      }
+
+      return stripeCapabilities;
+    }
+
+    try {
+      const contentfulCapabilities =
+        await this.capabilityManager.planIdsToClientCapabilities(
+          subscribedPrices
+        );
+
+      if (
+        isEqual(
```

Comment:
Does this check for deep equality? Meaning, will it check for equality in the value of the Object, i.e. the array of strings[]?

For example, take the objects below, will `isEqual` see them as equal or different?

```
const obj1 = { key1: ['val1', 'val2', 'val3']}

const obj2 = { key1: ['val3', 'val1', 'val2']}

isEqual(obj1, obj2) // is it true or false?
```

Reply:
Updated!